app/helpers/application_helper.rb

```diff
@@ -63,6 +63,12 @@ def field_or_none(label, value, tag: :p, tag_options: {}, separator: ': ',
                      label_class: label_class, value_class: value_class)
   end
 
+  def icon_link(url, icon, classes = '')
+    classes = 'grayscale' if url.blank?
```

Comment:
Need to handle the case:  icon is nil

suggestion:  the first statement in the method should be
```
  return '' if icon.nil?
```
(this is a _gating condition_)

I'm not sure if you should return an empty string or nil.